python/ray/util/placement_group.py

```diff
@@ -80,6 +84,30 @@ def _get_none_zero_resource(self, bundle: List[Dict]):
                 return key, value
         assert False, "This code should be unreachable."
 
+    def _fill_bundle_cache_if_needed(self):
+        if not self.bundle_cache:
+            # Since creating placement group is async, it is
+            # possible table is not ready yet. To avoid the
+            # problem, we should keep trying with timeout.
+            timeout_second = 5
```

Comment:
30 seconds? 5 is pretty short

Reply:
I think 5 seconds are reasonable. The table should be available before the placement group is actually created. When the creation requests are received from the GCS server, it flushes the record right away. So, it should mostly take < 1 second unless the GCS server is really busy (I think in this case, if it takes more than 5 seconds, the whole cluster is about to be crashed).

Reply:
Well, the kernel could start swapping or something; you never know. Please raise this to 30 seconds at least.

Reply:
Gotcha. Increased to 30 seconds!